app/ios/Podfile

```
@@ -27,16 +27,16 @@ if linkage != nil
-      auth_data.include?('Logged in to github.com account') &&
-        auth_data.include?('Git operations protocol: https')
-    rescue => e
-      puts 'gh auth status failed, assuming no HTTPS auth -- will try SSH'
-      false
-    end
+  begin
+    # backticks run command in shell and capture stdout, 2>&1 captures stderr as well
+    auth_data = `gh auth status 2>&1`
+    auth_data.include?("Logged in to github.com account") &&
+      auth_data.include?("Git operations protocol: https")
+  rescue => e
```

RuboCop (1.80.2)

[warning] Useless assignment to variable - `e`.

(Lint/UselessAssignment)